hw/ip/otbn/doc/dv/index.md

```diff
@@ -439,34 +461,39 @@ This instruction uses the `bnaq` encoding schema, with covergroup `enc_bnaq_cg`.
 This instruction uses the `bnaq` encoding schema, with an extra field not present in `bn.mulqacc`.
 Encoding-level coverpoints are tracked in covergroup `enc_bnaqw_cg`.
 
-Specific points as for `BN.MULQACC`, plus the generic flag group cover points.
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
```

Comment:
What do you mean by "See the accumulator overflow"? Also above and velow.

Reply:
We expect to see a `mulqacc.*` instruction that causes the (unsigned 256-bit) accumulator register to overflow when it is incremented. Is there another short formulation that you think would be clearer to someone who understands the ISA?